### PR TITLE
Add new event

### PR DESCRIPTION
**Pull request by fictionalparakeets**
In progress. Will be adding Modal rendering.
Re: Github Issue:
#10

### CODE REVIEW

File: src/components/Dashboard/NewEvent.jsx [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+      {formComplete ? (
+        <Redirect to="/dashboard" />
+      ) : (
+        <form autoComplete="off" onSubmit={(e) => e.preventDefault()}>
+          <div className="form-group">
+            <div>
+              <label htmlFor="event_title">Event Title:</label>
+              <input
+                type="text"
+                name="first_name"
```
[the remark this answers is not in the document]
Comment (fictionalparakeets) [reply src/components/Dashboard/NewEvent.jsx:77]: updated

File: src/components/Dashboard/NewEvent.jsx [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+                placeholder="X & X are tying the knot"
+                required
+                onChange={(e) => setEventTitle(e.target.value)}
+              />
+            </div>
+
+            <div>
+              <p>To Be Married:</p>
+              <label htmlFor="first_name" class="visually-hidden">
+                Person 1:
```
[the remark this answers is not in the document]
Comment (fictionalparakeets) [reply src/components/Dashboard/NewEvent.jsx:87]: Do we want the names to be optional? I'm confused if this has something to do with function or just a styling suggestion.

File: src/components/Dashboard/NewEvent.jsx [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+                name="email"
+                placeholder="email@email.com"
+                required
+                onChange={(e) => setEmail(e.target.value)}
+              />
+            </div>
+
+            <div>
+              <label htmlFor="phone">Phone: </label>
+              <input type="text" name="phone" required onChange={(e) => setPhone(e.target.value)} />
```
[the remark this answers is not in the document]
Comment (fictionalparakeets) [reply src/components/Dashboard/NewEvent.jsx:126]: For sure. I had purposely left all form types as simple as possible to make manual testing easier. I'll look into this when updating the form to use the calendar.
Thanks

File: src/components/Dashboard/NewEvent.jsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (SarahAvery) [reply src/components/Dashboard/NewEvent.jsx:87]: styling

**Review by SarahAvery — APPROVED** [review]

**Review by SarahAvery — APPROVED** [review]

**Pull request merged**